.env.example

```diff
@@ -15,8 +15,12 @@ POSTGRES_DATA=./db_data
 HAPI_SERVER_PORT=9090
 HAPI_SERVER_ADDRESS=0.0.0.0
 EOS_API_ENDPOINT=https://jungle.eosio.cr
-PROXY_INFO_CONTRACT_CODE=proxyaccount
-PROXY_INFO_CONTRACT_SCOPE=proxyaccount
+HAPI_PROXY_CONTRACT_CODE=proxyaccount
+HAPI_PROXY_CONTRACT_SCOPE=proxyaccount
+HAPI_STATS_CONTRACT_CODE=rateproducer
```

Comment:
@leisterfrancisco  

The values for  `HAPI_STATS_CONTRACT` and the `HAPI_RATING_CONTRACT` will always be the same 

I beleive we will be fine with just  `HAPI_RATING_CONTRACT`